example/src/MenuItems.constants.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Hyperlink } from '@edx/paragon';
+
+export const MENU_ITEMS = [
+  {
+    content: "Courses",
+    name: "courses",
+    destination: "#",
+    submenu: (
+      <div className="main-menu-courses">
+        <div className="d-sm-flex align-items center">
+          <h2 className="mt-2 mb-4 flex-grow-1">Courses by subject</h2>
+          <div><a href="#" className="btn btn-primary">All Subjects</a></div>
+        </div>
+        <ul className="list-unstyled mb-0">
```

Comment:
We'll need to figure out what the best way is to populate this list. It will probably involve making an API call to the discovery service backend to get the list of available subjects.